core/lib/mempool/src/mempool_store.rs

```diff
@@ -23,7 +23,7 @@ pub struct MempoolStats {
 #[derive(Debug)]
 pub struct MempoolStore {
     /// Pending L1 transactions
-    l1_transactions: HashMap<PriorityOpId, L1Tx>,
+    l1_transactions: BTreeMap<PriorityOpId, L1Tx>,
```

Comment:
please write a comment what operations you do on this field so that it's clear why this data structure fits well